nautilus_trader/core/nautilus_pyo3.pyi

```diff
@@ -628,6 +628,24 @@ class FuturesContract: ...
 class OptionsContract: ...
 class SyntheticInstrument: ...
 
+### Events
+class OrderDenied:
```

Comment:
I'm actually in two minds about this `pyi` file. Peoples auto imports are starting to pick this up and the pyo3 objects aren't compatible with legacy Cython. Thanks for including in your workflow for now though!